languages/cpp/reference/exercise-concepts/reverse-string.md

```diff
@@ -0,0 +1,77 @@
+# Reverse String
+
+## Task
+
+Reverse a string
+
+For example:
+input: "cool"
+output: "looc"
+
+
+## Example
+
+`reverse_string.h`
+
+```cpp
+#if !defined(REVERSE_STRING_H)
+#define REVERSE_STRING_H
+
+#include <string>
+
+namespace reverse_string
+{
+
+std::string reverse_string(const std::string& text);
+
+}
+
+#endif
+```
+
+`reverse_string.cpp`
+```cpp
+#include "reverse_string.h"
+
+namespace reverse_string
+{
+
+std::string reverse_string(const std::string& text)
+{
+    return std::string(text.rbegin(), text.rend());
+}
+
+}
+```
+
+## Concepts
+
+- .h file: the tests include `"reverse_string.h"`
+- .cpp file:  the most common approach is to use a `.cpp` for the definitions
+- namespaces: the function is member of a namespace
+- functions: used as the main entry point for the exercise
+- function arguments: the function has two arguments
```

Comment:
Nit. There're no two arguments in function ;)

Reply:
Yes, thank you, I have missed that =)